msm-sitemap.php

```diff
@@ -289,8 +289,11 @@ public static function get_post_year_range() {
 		global $wpdb;
 
 		$oldest_post_date_gmt = $wpdb->get_var( "SELECT post_date FROM $wpdb->posts WHERE post_status = 'publish' ORDER BY post_date ASC LIMIT 1" );
-		$oldest_post_year = date( 'Y', strtotime( $oldest_post_date_gmt ) );
-		$current_year = date( 'Y' );
+		
+		$oldest_post_year = $current_year = date( 'Y' );
```

Comment:
Returning false would cause an exception to be thrown in "check_year_has_posts" as it uses the return value of get_post_years_range directly in a foreach statement. I've updated the branch in the pull request so that an empty array is returned and will play nice with existing code and return values for the function.

Thoughts?